actions/app.go

```diff
@@ -29,31 +37,53 @@ func App() *buffalo.App {
 			},
 			SessionName: "_rdsapi_session",
 		})
-		// Automatically redirect to SSL
-		app.Use(forceSSL())
-
-		// Set the request content type to JSON
-		app.Use(middleware.SetContentType("application/json"))
 
 		if ENV == "development" {
-			app.Use(middleware.ParameterLogger)
+			app.Use(paramlogger.ParameterLogger)
+		}
+
+		// load json config
+		AppConfig, _ := common.LoadConfig("config/config.json")
```

Comment:
err?

Reply:
This was on purpose - I want to proceed even if the config is not loaded (useful when testing)

Reply:
log a warning?

Reply:
the `LoadConfig` is already logging before it returns an error
https://github.com/YaleSpinup/rds-api/blob/tg_prototype/pkg/common/config.go#L28